core/src/Cabana_VerletList.hpp

```diff
@@ -190,8 +187,20 @@ struct LinkedCellStencil
 //---------------------------------------------------------------------------//
 // Verlet List Builder
 //---------------------------------------------------------------------------//
+template <class Tag>
+struct CountNeighborsTag;
+
+template <>
+struct CountNeighborsTag<TeamVectorOpTag>
```

Comment:
Per my comments on looping below you might need an alias in here like `using tag = TeamVectorOpTag` to facilitate meta programming.